prive/datasets/examples/census.json

```diff
@@ -0,0 +1,95 @@
+[
```

Comment:
We should add the census to our tests once this PR is closed (i.e. once the tests are succeeding for texas with the new schema).

We should also think about where we want data (both csvs and jsons) to live in the file structure.